opam

```diff
@@ -20,7 +20,8 @@ remove: [["ocamlfind" "remove" "batteries"]]
 depends: [
   "ocamlfind" {>= "1.5.3"}
   "ocamlbuild" {build}
-  "qtest" {test & >= "2.0.0" & < "2.5"}
+  "qtest" {test & >= "2.0.0"}
+  "qcheck" {test & >= "0.6"}
```

Comment:
I didn't actually test this part of the change by having `opam` resolve dependencies.

You're right - `qtest < 2.5` are listed as conflicting with `qcheck` outright. I've inlined that constraint and just switched to `qtest >= 2.5`.